docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1960 @@
+{
```

Comment:
<p>Small preference: I would shows at.random.normal before pm.Normal.dist, just because we had been presenting Aesara operators until now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
I added this suggestion in https://github.com/pymc-devs/pymc/pull/5721/commits/af34f1b473de3e7f652116b446bdb9f65f9853f9 :)